src/components/containers/alert/Alert.specs.js

```diff
@@ -117,7 +118,7 @@ describe('when extraNotificationText is provided', () => {
     expect(instance.find('small').text()).toBe('test');
   });
 
-  it('displays a static icon', () => {
+  it('displays an icon', () => {
```

Comment:
Should we also test to see if the default icon is displayed, as well as a supplied icon?